### ISSUE TITLE
Contrast Issue of search button

### ISSUE THREAD

**Issue by ayanchoudhary** [cc-archive/vocabulary-legacy#476]
## Problem

![CC Search - Brave 17-06-2020 00_02_42](https://user-images.githubusercontent.com/45792768/84813848-3147e180-b02e-11ea-8e01-7363e5e55627.png)

## Description

Currently, this button has a contrast ratio of **3.45** whereas the desired minimum contrast ratio should be **4.5** or **AA/AAA large**

## Implementation

- [ ] I would be interested in implementing this feature.

**Comment by ayanchoudhary** [comment]
@panchovm @dhruvkb
